### ISSUE TITLE
Human Detection Project

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#7]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Human Detection Project</br>
:red_circle: **Aim** : The aim is to detect the human being using OpenCV with an accuracy over 90%.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/constantinwerner/human-detection-dataset</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by janeka1122** [comment]
Hi @abhisheks008, can you assign me this issue please. I am learning Deep Learning and Neural nets. I am fluent in Python. 

Full name : Chinmay Harkawat
GitHub Profile Link : https://github.com/janeka1122
Email ID : chinmayharkawatcodes1122@gmail.com
Participant ID (if applicable) : None that I know of 
Approach for this Project : Explore various algorithms and implement a Neural Net.
What is your participant role? SSOC (Social Summer of Code)

**Comment by abhisheks008** [comment]
Hey @janeka1122, thanks for showing interest in this project repo. I'll start assigning issues once all those formalities are done by the SSOC team. It's long a journey, don't worry about the deadline.

Best wishes 🚀

**Comment by 11Veeraj** [comment]
Hey @abhisheks008 can you assign me this issue? I'm from SSOC'22

**Comment by abhisheks008** [comment]
@janeka1122 do you wanna work in this issue?

**Comment by janeka1122** [comment]
Hi, thanks for the mention. Thing is the saw the rule about solving an issue in 6-7 days and I have an exam on 7th (Sunday)this week. 

So not sure if I'll be able to solve the issue on time.

**Comment by pandey-ankit-22** [comment]
Full name : Ankit Pandey 
GitHub Profile Link : https://github.com/pandey-ankit-22
Email ID : pandeyankit222003@gmail.com
Participant ID (if applicable): 
Approach for this Project : I will use multiple DL algorithm and NNs for this project.
What is your participant role?  Contributor (SSOC)

@abhisheks008 Please assign me this issue.

**Comment by abhisheks008** [comment]
Issue assigned to you @pandey-ankit-22. Make sure you complete this project before the deadline.

**Comment by achrekarom12** [comment]
Full name : Om Achrekar
GitHub Profile Link : achrekarom12
Email ID : [achrekarom@gmail.com](mailto:achrekarom@gmail.com)
Approach for this Project : Using various Image detection models such as VGG19, ImageNetV3 and Other suitable image detection model
What is your participant role? SSoC '23 Contributor

Please assign me this issue once my PR is merged.

**Comment by abhisheks008** [comment]
Issue assigned to you @achrekarom12
